src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.slf4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/dissolution")
+public class DissolutionController {
```

Comment:
Call this `HealthcheckController` - in the future we'll have Dissolution Applications that will likely utilise DissolutionController or something very similar. Best to not mix them up